ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -903,6 +903,51 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: aws-ipi-ovn-hypershift-fips-guest-f14
```

Comment:
Suggest to use `aws-ipi-ovn-hypershift-guest-fips-f14` for the test name. 

The `aws-ipi-ovn-hypershift-guest` comes from the workflow, which instruct us to search the definition in folder hierarchy `cucushift/installer/rehearse/aws/ipi/ovn/hypershift/guest` when we found something not working as expected, adding fips in the middle break this rules.

Reply:
For hypershift guest or mgmt, it's a type of testing. For example, for the `guest` scenario, we run e2e tests on the HostedCluster. For the `mgmt` scenario, we run e2e tests on mgmt cluster. `aws-ipi-ovn-hypershift-fips` is an integrated entity, so I believe the current approach is better.

Reply:
`mgmt` is a different situation, as mgmt is not part of the workflow name.

Reply:
@liangxia As I know, we will not create separate folder (like FIPS, Private, Privatelink etc) under mgmt or guest. So, I do not see we need to adjust the rules in future (I know current name rule is following folder's structure to give name). In addition, we will easier to share our test result to Dev and PM with integrated entity name like aws-ipi-ovn-hypershift-fips or aws-ipi-ovn-hypershift-private etc as search key words in prow ci. And it will be easier for them to see both mgmt and guest test result.

Reply:
As HyperShift will support ROSA / ARO gating test as well, we should make it easier for stakeholders to search with key words and see the result for both mgmt and guest.

Reply:
FIPS and so on are controlled via env var, it's not part of the workflow name, so we always append them to the workflow name.
All jobs can be searched/shared with regexp. E.g, for [hypershift fips](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/?repo=openshift%2Fopenshift-tests-private&type=periodic&job=*hypershift*fips*), use `*hypershift*fips*`

Reply:
@liangxia Simple logic for hypershift is: When we put aws-ipi-ovn-hypershift-fips, we can find both aws-ipi-ovn-hypershift-fips-mgmt and aws-ipi-ovn-hypershift-fips-guest in the same time. If you put aws-ipi-ovn-hypershift-private, we can find aws-ipi-ovn-hypershift-private-mgmt and aws-ipi-ovn-hypershift-private-guest. mgmt and guest always show to stackholders in pair.

Reply:
When hypershift QE built pipeline, he will always consider to add for both mgmt and guest both. This is one group and always show result as group.

Reply:
@liangxia Do you have any other concern about merge this PR?

Reply:
![image](https://github.com/openshift/release/assets/4716987/9399f715-d2a9-4db9-9055-0e2bbc881157)